agimus_demo_05_pick_and_place/agimus_demo_05_pick_and_place/hpp_client.py

```diff
@@ -109,6 +119,7 @@ def set_problem(self):
         self.ps.selectPathValidation("Graph-Progressive", 0.01)
 
         vf = ViewerFactory(self.ps)
+        self.vf = vf
```

Comment:
```suggestion
        self.vf = ViewerFactory(self.ps)
```

Reply:
Feel free to take the update the code as you please. I didn't want to change the usage of `vf` below, hence my initial change.